ci/recipe/meta.yaml

```diff
@@ -32,6 +32,8 @@ requirements:
     - q2templates {{ release }}.*
     - q2-sample-classifier {{ release }}.*
     - q2-feature-table {{ release }}.*
+    - matplotlib = 3.2.2
+    - matplotlib-base = 3.2.2
```

Comment:
```suggestion
    - matplotlib 3.2.2
    - matplotlib-base 3.2.2
```

Reply:
Do you think we should do an exact pin like this, @nbokulich? Or should we do something like "less than 3.3"?

Reply:
hmm... better do a "less than" just in case, seems less likely to clash with other plugins.